R/ppmify.R

```diff
@@ -302,6 +305,36 @@ grid <- function (area, density) {
 
 }
 
+bias <- function (area, bias_points, offset.raster=FALSE){
+  
+  # calculate weights for integration points when using pseudo-observations as integration points
+  # weight = total study area / number of integration/quadrature points
+  
+  if (offset.raster){ # if offset raster is also provided, calculate sum of raster values
+    
+    a <- cellStats(area, sum, na.rm=TRUE)  
```

Comment:
Ah, that makes sense! Cool, so the algorithm's doing the right thing. As you say, a different name might be sensible.

How about you remove `biasRaster` as a method, and instead add a flag allowing he user to say that what's passed to the `area` argument is already pre-loaded with a meaningful effort metric. Then that flag can be passed to each method (grid, count and bias_points) and used when doing extraction to calculate weights?